Resources/Prototypes/_RMC14/Entities/Objects/xeno_figurines.yml

```diff
@@ -0,0 +1,43 @@
+# these dont get spawned by patron figurine spawner since no PatronFigurineComponent
+# map in "xeno figurine spawner" or spawn them midround :)
+
+- type: entity
+  abstract: true
+  parent: RMCBaseFigurinePatron
```

Comment:
I got spooked by this but apparently this is fine
Should probably rename it to not be FigurinePatron some day

Reply:
yes the base figure actually has no patron specific stuff